src/containers/FormContainer.js

```diff
@@ -0,0 +1,70 @@
+import React, {Component} from "react";
+import {debounce} from "throttle-debounce";
+import {connect} from "react-redux";
+// import {addUserName} from "../actions/addUsername";
+import {setResponse} from "../actions/setResponse";
+import {Request} from "projects/javascript/Request.js";
+import ContentPresenter from "../components/ContentPresenter";
+import I from "immutable";
+
+
+function mainApi(input = "nzakas") {
+  let url = [];
```

Comment:
because the datas which i need are coming from different url's, i've requested to the server with an array of urls.